src/main/resources/standardTestCases/local_gcs/local_gcs.wdl

```diff
@@ -0,0 +1,34 @@
+task makeGcsFile {
+  command {
+    echo "I am writing stuff" > out
+  }
+
+  runtime {
+    docker: "ubuntu:latest"
+  }
+
+  output {
+    File outputFile = "out"
+  }
+}
+
+task readGcsFileFromLocal {
+  File gcsFile
+
+  command {
+    cat ${gcsFile}
+  }
+
+  runtime {
+    backend: "Local"
+  }
+
+  output {
+    String result = read_string(stdout())
+  }
+}
+
+workflow local_gcs {
+  call makeGcsFile
```

Comment:
can this be something like `makeGcsFileInJES` to be more explicit as to what's happening?